content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,46 @@ menu:
 weight: 101
 ---
 
+## v2.0.0 General Availability [2020-11-09]
```

Comment:
@russorat, anything else we'd like to highlight about 2.0 GA in release notes? Thank you!

Reply:
these won't be final until the tag